test/unit/components/response-body.jsx

```diff
@@ -28,25 +28,33 @@ describe("<ResponseBody />", function () {
   it("renders ResponseBody as 'image/svg'", function () {
     props.contentType = "image/svg"
     const wrapper = shallow(<ResponseBody {...props} />)
-    console.warn(wrapper.debug())
     expect(wrapper.find("highlightCodeComponent").length).toEqual(0)
   })
 
   it("should render a copyable highlightCodeComponent for text types", function () {
     props.contentType = "text/plain"
     props.content = "test text"
     const wrapper = shallow(<ResponseBody {...props} />)
-    console.warn(wrapper.debug())
     expect(wrapper.find("highlightCodeComponent[canCopy]").length).toEqual(1)
   })
 
-  it("should render Download file link for non-empty response", function () {
+  it("should render Download file link for non-empty Blob response", function () {
     props.contentType = "application/octet-stream"
     props.content = new Blob(["\"test\""], { type: props.contentType })
     const wrapper = shallow(<ResponseBody {...props} />)
     expect(wrapper.text()).toMatch(/Download file/)
   })
 
+  it("should render Download file link for non-empty text response", function () {
```

Comment:
Edited the file to provide proper test that is passing against the changes in `src/